src/lib.rs

```diff
@@ -1056,10 +1028,6 @@ impl PublicKey {
             result_glwe = self.glwe_sum(&result_glwe, &many_lut[i].0);
         }
```

Comment:
Why the rotation of half_box_size is removed here ? 

        // Rotation of -half_box_size to get the accurate result when we sample extract
        let poly_monomial_degree = MonomialDegree(2 * ctx.polynomial_size().0 - ctx.box_size() / 2);
        self.glwe_absorption_monic_monomial(&mut result_glwe, poly_monomial_degree);

Reply:
because `to_many_lut` has been changed to use `LUT::from_lwe` which centers the boxes already